### ISSUE TITLE
activation function

### ISSUE THREAD

**Issue by jiexiabalaa** [jiexiabalaa/tensorflow-minist#2]
#sigmod
import tensorflow as tf
a = tf.constant([[1.0, 2.0], [1.0, 2.0], [1.0, 2.0]])
sess = tf.Session()
print(sess.run(tf.sigmoid(a)))
#relu & softplus
a = tf.constant([-1.0, 2.0])
with tf.Session() as sess:
    b = tf.nn.softplus(a)
    print(sess.run(b))
#dropout
a = tf.constant([[-1.0, 2.0, 3.0, 4.0]])
with tf.Session() as sess:
    b = tf.nn.dropout(a, 0.5, noise_shape = [1,4])
    print(sess.run(b))
    b = tf.nn.dropout(a, 0.5, noise_shape = [1,1])
    print(sess.run(b))
